src/TLuaInterpreter.cpp

```diff
@@ -820,29 +815,27 @@ int TLuaInterpreter::wrapLine(lua_State* L)
 {
     int s = 1;
     int n = lua_gettop(L);
-    string a1 = "main";
+    string window = "main";
     if (n > 1) {
```

Comment:
Since this is the only use of `n` (I think) we might just as well test `lua_gettop(L)` directly in the `if(`...`)` I guess.

Reply:
Updated